site/content/ytt/docs/latest/data-values-schema-migration-guide.md

```diff
@@ -153,33 +152,38 @@ Now simply follow the steps in either of the previous examples to migrate the pr
 ---
 
 ## How do I provide default values for an array?
-Arrays in Schemas have [special behavior](lang-ref-ytt-schema.md#defaults-for-arrays):
+Arrays in Schemas are [handled differently](lang-ref-ytt-schema.md#defaults-for-arrays) than other types:
 exactly one element is specified in the array, and that value is _only_ used to infer the type of that array's elements —
 the default value for arrays is always empty.
```

Comment:
'always' seems a little strong now that we have the annotation to change the default value. It's technically not incorrect, but I see it used in how-to-write-schema.md:106 as well and I wonder if this language could discourage the reader from continuing further?

Reply:
nice catch; I absolutely agree. It would be better if we simply say that the default value is not taken from the declaration, but is empty list... that a default value is given through the annotation. 👍🏻 

On it.

Reply:
@gcheadle-vmware I pushed some updates... how do you think the revision reads?